test/test-ffmpegCommandService.js

```diff
@@ -48,7 +48,7 @@ describe('FfmpegCommandService', function() {
                     workingDirectory: Path.resolve(__dirname, './fixtures/assets1')
                 });
 
-                Should(command).eql('/usr/local/bin/ffmpeg -i /Users/jonathanfulton/projects/misc/whipple-video/test/fixtures/assets1/sample.mp4 -filter_complex " [0:v] trim=start=0:duration=600, setpts=PTS-STARTPTS [v0]; [v0] concat=n=1:v=1:a=0 [v_concat]" -map "[v_concat]" -y /Users/jonathanfulton/projects/misc/whipple-video/test/fixtures/assets1/output.mp4');
+                Should(command).eql('/usr/local/bin/ffmpeg -i ' + __dirname + '/fixtures/assets1/sample.mp4 -filter_complex " [0:v] trim=start=0:duration=600, setpts=PTS-STARTPTS [v0]; [v0] concat=n=1:v=1:a=0 [v_concat]" -map "[v_concat]" -y ' + __dirname + '/fixtures/assets1/output.mp4');
```

Comment:
It's probably worth factoring out the paths into variables and using Path.resolve() along with \`{var}\`

Reply:
Good call! I refactored to use `Path.resolve()` all around. I opt'ed to not do ES6 string interpolation at this time though as it would require manual edits in 36 places with a tiny small gain in code quality.